### DISCUSSION TITLE
Does wotlk wowhead classic version show now correct loot chances values ?

### DISCUSSION THREAD

**Discussion by CraftedRO** [TrinityCore/TrinityCore#28264] [Q&A] [answered]
Does https://www.wowhead.com/wotlk/npc=loot#drops show now correct percent values for loot drops that shoud be implemented in TC 3.3.5 ?
for example :

![drops](https://user-images.githubusercontent.com/24683355/191184228-6b87384a-75ae-46a2-83ea-02d010c5f547.PNG)

I want to start adding some loot tables now since we got official database up.

**Comment by Shauren** [comment, marked as answer]
No, of course not, they are based on statistics collected by players

**Comment by funjoker** [comment]
Take MobInfo2 change settings so it collects grey items as well.
Now the problem. If you kill creatures which drop quest items is counts these too.
But quest items usually only drop when quest is active.
So the % for quest items is broken witht this addon :D.

**Reply by Faq** [reply]
I guess, addon could be modified to not count qitems?

**Reply by funjoker** [reply]
Maybe. Would need a list of quest items maybe ?
